translations/en.json

```diff
@@ -607,6 +614,15 @@
             "message1": "Do you want to keep this draft or delete it?"
         }
     },
+    "modals": {
+        "apply_filters": {
+            "button_apply": "Apply",
+            "label_confirm_request": "Do you want to apply the filter?",
+            "label_involved_messages": "<0>{{involvedMessagesCount}} messages</0> will be processed inside the selected folder.",
+            "title": "Application filter \"{{filterName}}\""
+        }
+    },
+    "nameTitle": "",
```

Comment:
I assume this has been generated by the watch mode and should be removed

Reply:
Yup. I remove it right away